CONTRIBUTING.md

```diff
@@ -155,6 +155,35 @@ We use [Milestones](https://github.com/bevyengine/bevy/milestones) to track issu
 
 There are also two priority labels: [`P-Critical`](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AP-Critical) and [`P-High`](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AP-High) that can be used to find issues and PRs that need to be resolved urgently.
 
+### Closing PRs and Issues
+
+From time to time, PRs are unsuitable to be merged in a way that cannot be readily fixed.
+Rather than leaving these PRs open in limbo indefinitely, they should simply be closed.
+
+This might happen if:
+
+1. The PR is spam or malicious.
+2. The work has already been done elsewhere or is otherwise fully obsolete.
+3. The PR was successfully adopted.
+4. The work is particularly low quality, and the author is resistant to coaching.
+5. The work adds features or abstraction of limited value, especially in a way that could easily be recreated outside of the engine.
+6. The work has been sitting in review for so long and accumulated so many conflicts that it would be simpler to redo it from scratch.
+7. The PR is pointlessly large, and should be broken into multiple smaller PRs for easier review.
+
+PRs that are `S-Adopt-Me` should be left open, but only if they're genuinely more useful to rebase rather than simply use as a reference.
+
+There are several paths for PRs to be closed:
+
+1. Obviously, authors may close their own PRs for any reason at any time.
+2. If a PR is clearly spam or malicious, anyone with triage rights is encouraged to close out the PR and report it to Github.
+3. If the work has already been done elsewhere, adopted or otherwise obsoleted, anyone with triage rights is encouraged to close out the PR with an explanatory comment.
+4. Anyone may nominate a PR for closure, by bringing it to the attention of the author and/or one of the SMEs / maintainers. Let them press the button, but this is generally well-received and helpful.
```

Comment:
```suggestion
4. Anyone may nominate a PR for closure, by bringing it to the attention of the author and / or one of the SMEs / maintainers. Let them press the button, but this is generally well-received and helpful.
```

One slash has spaces around it, the other does not. I don't have a strong preference, but consistency is preferable :)